public/hello.html

```diff
@@ -0,0 +1,132 @@
+      </svg>
+      <h2>Шоурум и интернет-магазин предметов интерьера и декора</h2>
+      <hr class="separator"></hr>
+      <ul class="list">
+        <li>
+          <a class="link" href="https://weareforfolks.com">weareforfolks.com</a>
+          <p class="text">Наш каталог, рекомендации и анонсы свежих поступлений</p>
+        </li>
```

Comment:
IMHO it would be better to make `li` like this:
```
    <p class="text">Наш каталог, рекомендации и анонсы свежих поступлений:</p>
    <a class="link" href="https://weareforfolks.com">weareforfolks.com</a>
```